recipes/gromacs/2024/gh200-mpi/environments.yaml

```diff
@@ -0,0 +1,34 @@
+gromacs:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@12.3
+  unify: when_possible
+  specs:
+  - cmake
+  - cuda@12.4
+  - blas
+  - lapack
+  - hwloc
+  - fftw
+  - python@3.10
+  - gromacs@2024.1 +cuda +mpi +hwloc
+  - gromacs@2022.5 +plumed +cuda +hwloc
```

Comment:
Two comments on this:
1. Would it be better having two different environments and use `unify: true`?
2. While this has been made clear in the documentation, it might be  confusing to have a `2022.5` version in the same uenv. Would it be better to have a different uenv providing `2022.5` (with `plumed`)? Or is this mix of versions part of a well defined workflow users are used to?